### ISSUE TITLE
Weird food item behaviour while stored in disconnected fridge

### ISSUE THREAD

**Issue by silvadomi** [CleverRaven/Cataclysm-DDA#74594]
### Describe the bug

On a new playtest with 0.H, I started on an apartment tower. Went to one of the apartments, checked the fridge, there's butter, cheese, salsa, displayed in the inventory screen (pickup) together in one entry, as expected. Went around the building a bit, and when I was coming back to that fridge in the evening, the same food items had split up to be displayed separately in seemingly random groups (see butter, cheese and salsa entries in the screenshot below).

Also, I didn't have a screenshot of that, but the cheese strangely was labelled as old when I first saw it, but when I ate one piece in the evening, it was not old anymore.

### Attach save file

[Heavenly Chimney-trimmed.tar.gz](https://github.com/user-attachments/files/15857950/Heavenly.Chimney-trimmed.tar.gz)

### Steps to reproduce

1. Spawn a new character in a residential area.
2. See what's in the fridge, make sure you have some items spawning together in containers (paper wrapper, plastic bag, jar...).
3. Move around until evening, maybe eat one or two of the items.
4. Go to pickup food items, see that they are separated in the display window.

### Expected behavior

Food items that spawn together in one container and are left in the same place the whole time should not be separated in inventory display screens.

### Screenshots

![food-items](https://github.com/CleverRaven/Cataclysm-DDA/assets/171120271/80bb8baf-56e0-4636-9c15-8a8fb2a5504b)

### Versions and configuration

- OS: MacOs
    - OS Version: Name: macOS; Version: <unknown>; Build: <unknown>; 
- Game Version: 0.H-201-gbe7a566ffc [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by DalenTalas** [comment]
I suspect this is expected behavior relating to the spoilage mechanic. The "disassociated" items have slightly different time-to-live, and thus are not grouped together.

**Comment by silvadomi** [comment]
@DalenTalas yes I also thought so. I am mostly wondering about, and wouldn't say it's expected behavior, that items which spawned in a combined stack and spent their whole lifetime as a combined stack would be able to acquire different times to spoil at some point.

Might be worthwhile to check whether the same also occurs for items that are harvested or crafted instead of spawned on map generation.
